src/main/java/sirius/biz/tenants/TenantUserManager.java

```diff
@@ -776,7 +776,11 @@ protected UserSettings getUserSettings(UserSettings scopeSettings, UserInfo user
 
         return configCache.get(user.getUniqueName(), i -> {
             Config cfg = scopeSettings.getConfig();
-            cfg = tenantConfig.withFallback(cfg);
+
+            if (tenantConfig != null) {
+                cfg = tenantConfig.withFallback(cfg);
```

Comment:
finde man könnte dabei auch kurz den codestyle verbessern indem man cfg und i durch sprechende variablen namen ersetzt

Reply:
i --> ignored

Reply:
Dann kann man ja ignored hinschreiben :) `i` als Variablenname steht ja in 99% der fälle für was anderes